app/routes/page.php

```diff
@@ -153,11 +209,12 @@ function renderGuide(Slim $app, Guide $guide, Category $category)
     }
 
     if ($fetchContent) {
-        $markdown = file_get_contents('https://raw.githubusercontent.com/piwik/piwik/master/CHANGELOG.md');
+        $markdown = file_get_contents('https://raw.githubusercontent.com/piwik/piwik/3.x-dev/CHANGELOG.md');
```

Comment:
Could this URL be constructed dynamically reusing the constant `LATEST_PIWIK_DOCS_VERSION`  maybe?